app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Idv
+  class AamvaStateMaintenanceWindow
+    class << self
+      def in_maintenance_window?(state)
+        return false unless (window = window_for_state(state))
+
+        Time.use_zone(window['tz']) do
+          start_time = Time.parse("today #{window['start_time']}")
```

Comment:
I'm curious if there's a better way of doing this. I spent a while searching last night and did not find one.

Reply:
1. Are there any maintenance windows that cover midnight? ex 11pm to 5am? They might have some trouble with an approach like this
2. Another option could be `Time#change`, but it's roughly the same idea

```
start_time = Time.zone.now.change(hours: 1, minutes: 1)
```